src/README.md

```diff
@@ -0,0 +1,60 @@
+# DMD's source code
+
+This directory contains the source code of the dmd compiler.
+Refer to [this file](dmd/README.md) for an overview of the actual source files.
+
+## Building the compiler
+
+The compiler can be built using the integrated build system `build.d`.
+
+
+```console
+./build.d
+```
+
+See the output of `./build.d --help` to get a list of all supported targets
+or see the list of major targets below.
+
+### Bootstrapping
+
+You can also bootstrap using `bootstrap.sh` if you don't currently have a D
```

Comment:
In general, do not use "you" in directions. For example,

"Use `bootstrap.sh` to bootstrap the compiler if a D compiler is not currently installed."